styles.css

```diff
@@ -0,0 +1,132 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background: "#e5e5e5";
+}
+
+.container {
+  padding: 0 5rem;
+}
+
+nav {
+  justify-content: space-between;
+  height: 72px;
+  background-color: #ffff;
+  width: 100%;
+}
+
+#menu {
+  display: none;
+}
+
+ul {
+  display: flex;
+  flex-direction: row;
+  gap: 1rem;
+}
+
+.flexr {
+  display: flex;
+  flex-direction: row;
+  gap: 1rem;
+}
+
+.flexc {
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+}
+
+li {
+  list-style: none;
+}
+
+.link {
+  text-decoration: none;
+  color: #6070ff;
+  font-size: 1.5rem;
+  font-weight: 600;
+}
+
+#logo {
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  margin: auto 0;
+}
+
+#header {
+  background-image: url("/images/header-background-mobile.png");
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  height: calc(100vh - 72px);
+  border-bottom: 5px solid #e5e5e5;
+  border-radius: 0 0 0 100px;
+  background-color: #ffff;
+}
+
+section {
+  margin: auto 0;
+  max-width: 70%;
+}
+
+section h1 {
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #172b4d;
+}
+
+section p {
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+}
+
+section h3 {
+  font-family: Poppins, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+}
+
+div .flexr {
+  gap: 2rem;
+}
+
+@media screen and (width <= 768px) {
+  .container {
+    padding: 0 1rem;
+  }
+
+  #menu {
+    display: block;
+    margin-top: auto;
+    margin-bottom: auto;
+  }
+
+  ul {
+    display: none;
+  }
+
+  section {
+    margin: auto 0;
+  }
+}
```

Comment:
Please remove these screen size for mobile since we are always advised to apply the mobile-first approach when building our website as it is a very useful way to ensure the responsiveness of our websites 😄